### PR TITLE
Jcm docs structure

### PR DESCRIPTION
**Pull request by cmcfadzean**

### CODE REVIEW

**Review by pixelflips — APPROVED** [review]
👍 +1

File: app/views/sage/application/_assistant.html.erb [hunk 1]
```diff
@@ -6,5 +6,7 @@
   <%= image_tag("sage/sage.svg", class: "sage-assistant__branding")%>
   <div class="sage-assistant__body">
   </div>
-  <div class="sage-assistant__actions"></div>
+  <div class="sage-assistant__actions">
+    <h4 class="version-number">v<%= sage_version %></h4>
```
Comment (philschanely) [inline app/views/sage/application/_assistant.html.erb]: What do you think about making this a `p` and adding `t-sage-body-small-bold`?

**Review by philschanely — APPROVED** [review]
One non-blocking suggestion.

**Review by teenwolfblitzer — APPROVED** [review]
👍 LGTM

**Pull request merged**